core/src/main/scala/org/apache/spark/rdd/BlockRDD.scala

```diff
@@ -47,7 +47,9 @@ class BlockRDD[T: ClassTag](sc: SparkContext, @transient val blockIds: Array[Blo
     blockManager.get[T](blockId) match {
       case Some(block) => block.data.asInstanceOf[Iterator[T]]
       case None =>
-        throw new Exception("Could not compute split, block " + blockId + " not found")
+        throw new Exception(
+          s"Could not compute split, block $blockId not found of RDD $id not found")
```

Comment:
The first 'not found' is redundant and should be removed. This can go on one line as before